src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+        if (user != null) {
+            return (user.getPassword().equals(password));
+        }
         return false;
```

Comment:
condition can be compressed 

```suggestion
        return user != null && user.getPassword().equals(password)
```